Week2/121_Best_Time_To_buy_Sell_Stocks.cpp

```diff
@@ -0,0 +1,35 @@
+class Solution {
+    const int INF = 1000 * 1000 * 1000;
+    
+public:
+    int maxProfit(vector<int>& prices) {
+        // We move from the beginning to the end with one for loop.
+        // At each iteration we try to greedily update the answer.
```

Comment:
Next time, please, include details on how you would do it. Too abstract comment.